src/include/compat.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
```

Comment:
Even though the header's very short, please include a copyright notice.

Reply:
Uses the same license header as `maths_utils.c` now.